wp-content/themes/haruoshi_origin/archive-news.php

```diff
@@ -10,6 +10,21 @@
 <article class="container">
     <section class="news section-wrapper">
         <h1 class="section-title">News</h1>
+        <?php
+        $args = array(
+            'orderby' => 'name',
+            'order' => 'ASC'
+        );
+        $taxonomy_terms = get_terms('news_tag');
+        if ($taxonomy_terms):?>
+        <ul class="news__tag-list">
+            <?php foreach( $taxonomy_terms as $term ):?>
+            <li class="news__tag">
+                <a href="<?= get_term_link( $term ) ?>">#<?= $term->name ?></a>
```

Comment:
こうすればわざわざtaxonomy-news_tag.phpを作らなくていい気がするんだけどどう？
```suggestion
                <a href="<?= get_post_type_archive_link("news") . "?" . http_build_query(["news_tag" => $term->slug]); ?>"></a>
```

Reply:
ほんまや

Reply:
done https://github.com/oooshima/origin_wordpress/pull/34/commits/53cb043f011e9403935e5e194a9ce9db3c6d36a7